File: array_linearSearch.cpp
#include <iostream>
using namespace std;

int main() {

    int arr[] = {100, 200, 300, 400, 500};
    int size = 5;

    int target = 500;

    for (int i = 0; i < size; i++) {

        if (arr[i] == target) {
            cout << "Element found at index: " << i << endl << "And the value is :"<< arr[i];
            return 0;
        }
    }

    cout << "Element not found";

    return 0;
}

File: array_linearSearch.js
const arr = [100, 200, 300, 400, 500];
const target = 500;

let found = false;

for (let i = 0; i < arr.length; i++) {
    if (arr[i] === target) {
        console.log(`Found at index ${i}`);
        found = true;
        break;
    }
}

if (!found) {
    console.log("Element not found");
}